### PR TITLE
Hybrid AAD was not listed

### PR DESCRIPTION
**Pull request by joinimran**
I have added Hybrid AAD resource information. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2111

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md [hunk 1]
```diff
@@ -90,3 +90,5 @@ In environments that have more restrictive internet access, or for those that re
     -   <https://support.office.com/en-us/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2> (includes all Office services, DNS names, IP addresses; includes Azure AD and other services that may overlap with those listed above)
 
 -   **Certificate revocation lists (CRLs).**  Some of these services will also need to check certificate revocation lists (CRLs) for certificates used in the services.  A full list of these is documented in the Office documentation at <https://support.office.com/en-us/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2#bkmk_crl> and <https://aka.ms/o365chains>.
+
+-   **Hybrid AAD Join.**  Hybrid AAD can be join, the machine should on corporate network for hybrid ADD join to work. See documentation at <https://docs.microsoft.com/en-us/windows/deployment/windows-autopilot/user-driven-hybrid>
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md]:
```suggestion
-   **Hybrid AAD Join.**  Hybrid AAD can be joined, but the device must be on a corporate network for hybrid ADD join to work. See [Windows Autopilot user-driven mode for hybrid Azure Active Directory join] <https://docs.microsoft.com/windows/deployment/windows-autopilot/user-driven-hybrid> for more information.
```

**Comment by joinimran** [comment]
Commit the changes.

**Comment by mypil** [comment]
@DulceMontemayor - Can you please assist to check, approve and merge the content of this PR based on issue #2111?

Thank you.

**Comment by mypil** [comment]
@DulceMontemayor - We would like to follow up on this. Can you please assist to check, approve and merge the content of this PR based on issue #2111?

Thank you.

**Comment by mypil** [comment]
@DulceMontemayor @greg-lindsay - We would like to follow up on this. Can you please assist to check, approve and merge the content of this PR based on issue #2111?

Thank you.

**Comment by mypil** [comment]
@DulceMontemayor @greg-lindsay @AndreaBarr - Can you help review and merge this PR? We noticed that there is a conflict but @joinimran could not seem to resolve the branch conflict and we found that the content source gets a 404 not found error. Any thoughts on how we can resolve this?

Thank you.

**Comment by joinimran** [comment]
@AndreaBarr Grammar correction applied.

File: windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md [hunk 2]
```diff
@@ -90,3 +90,5 @@ In environments that have more restrictive internet access, or for those that re
     -   <https://support.office.com/en-us/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2> (includes all Office services, DNS names, IP addresses; includes Azure AD and other services that may overlap with those listed above)
 
 -   **Certificate revocation lists (CRLs).**  Some of these services will also need to check certificate revocation lists (CRLs) for certificates used in the services.  A full list of these is documented in the Office documentation at <https://support.office.com/en-us/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2#bkmk_crl> and <https://aka.ms/o365chains>.
+
+-   **Hybrid AAD Join.**  Hybrid AAD can be join, the machine should be on corporate network for hybrid ADD join to work. See details at <https://docs.microsoft.com/en-us/windows/deployment/windows-autopilot/user-driven-hybrid>
```
Comment (DaniHalfin) [inline windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md]: typo here (ADD = AAD). also, please remove locale on the URL.

**Comment by DaniHalfin** [comment]
@greg-lindsay is the content owner here. He'll need to sign-off.

File: windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md (hunk 2 above)
Comment (joinimran) [reply windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md]: Fixed this issue.

**Comment by joinimran** [comment]
As there is a issue with the branch so I have created a new PR at https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4035 which should be considered as the replacement of this current PR.

**Comment by mypil** [comment]
@joinimran - Please close this PR.

**Pull request closed without merging**